f5_openstack_agent/lbaasv2/drivers/bigip/test/test_vcmp.py

```diff
@@ -126,9 +126,9 @@ def old_setup_vcmp_method_test(mock_mgmt_root, mock_driver, mock_host_obj):
 def test___init__(setup_vcmp_mgr):
     vcmp = setup_vcmp_mgr
     assert vcmp.vcmp_hosts != []
-    assert len(vcmp.vcmp_hosts) is 2
+    assert len(vcmp.vcmp_hosts) == 2
```

Comment:
these are common places which IMO is irrelevant to queens_engineering. put them into a separate commit so that they are submitted into each branch we need?(e.g. 9.6 9.7 etc, if needed).

Reply:
I will do it, after I fix the loopback4 things, or you can help me a bit :D